source/getting-started/use-cases/configuration-assessment.rst

```diff
@@ -81,43 +81,35 @@ The Wazuh dashboard has a **Configuration Assessment** module that allows you to
+The image below shows the policy based on the CIS benchmark for Ubuntu Linux 22.04. You can see that 191 checks were run against the Ubuntu 22.04 endpoint. Out of these, 56 passed, 87 failed, and 48 are not applicable to the endpoint. It also shows a score of 39% which is calculated based on the number of tests passed.  
 
 .. thumbnail:: /images/getting-started/use-cases/sca/cis-benchmark-ubuntu22-results.png
    :title: Results for CIS benchmark for Ubuntu 22.04 checks
    :alt: Policy for CIS benchmark for Ubuntu 22.04 checks
    :align: center
    :width: 80%
 
-You can click on the checks to get more information. In the image below, you can see information such as rationale, remediation, and a description of the check with ID ``28500``.
+You can click on the checks to get more information. In the image below, you can see information such as rationale, remediation, and a description of the check with ID ``3003``.
 
-.. thumbnail:: /images/getting-started/use-cases/sca/check-28500-results.png
-   :title: Results for CIS benchmark for Ubuntu 22.04 check ID 28500
-   :alt: Results for CIS benchmark for Ubuntu 22.04 check ID 28500
+.. thumbnail:: /images/getting-started/use-cases/sca/check-3003-results.png
+   :title: Results for CIS benchmark for Ubuntu 22.04 check ID 3003
+   :alt: Results for CIS benchmark for Ubuntu 22.04 check ID 3003
    :align: center
    :width: 80%
 
-The above SCA scan result shows ``failed`` because the ``/tmp`` directory is not on a separate partition and the directory is not mounted at boot time. If the remediation is implemented, the result will change to ``passed`` thereby improving the security of the endpoint.
+The above SCA scan result shows ``failed`` because the public key authentication for ssh is not enabled. If the remediation is implemented, the result will change to ``passed`` thereby improving the security of the endpoint.
 
 Implementing SCA remediation steps
 ----------------------------------
 
-In the example in the previous section, implementing the remediation provided by the Wazuh SCA improves the security of the endpoint. This involves mounting the ``/tmp`` directory to a separate partition and adding some options such as ``nodev``, ``noexec``, and ``nosuid`` to the mount point. In the image below, you can see the status of the checks ``28500``–``28503`` have changed to ``passed``.
+In the example in the previous section, implementing the remediation provided by the Wazuh SCA improves the security of the endpoint. This involves changing the ``PublkeyAuthentication`` option value in the ``sshd_config`` file. In the image below, you can see the status of the check ``3000`` have changed to ``passed``.
```

Comment:
**nitpick**: typos:
- `PublkeyAuthentication`: `PubkeyAuthentication`
- `3000`: `3003` (check id displayed in the previous section)
- `have`: `has`

```suggestion
In the example in the previous section, implementing the remediation provided by the Wazuh SCA improves the security of the endpoint. This involves changing the ``PubkeyAuthentication`` option value in the ``sshd_config`` file. In the image below, you can see the status of the check ``3003`` has changed to ``passed``.
```

Reply:
Done